server/testutils/factories/cas1NonArrival.ts

```diff
@@ -0,0 +1,12 @@
+import { Factory } from 'fishery'
+import { Cas1NonArrival } from '@approved-premises/api'
+import { faker } from '@faker-js/faker'
+import { DateFormats } from '../../utils/dateUtils'
+
+export default Factory.define<Cas1NonArrival>(() => ({
+  reason: faker.word.words(3),
```

Comment:
Is this supposed to be an id? If so maybe using `faker.string.uuid()` might be more appropriate.

Reply:
Oops - yes, it's an ID, not the reason name. Good spot!